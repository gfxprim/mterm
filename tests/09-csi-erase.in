10 10
this is\n
very long\n
text that\n
is filling\n
buffer\n
\e[2J\e[H
this is\n
very long\n
text that\n
showing\n
the buffer\n
