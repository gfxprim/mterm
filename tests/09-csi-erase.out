 ----------
|this is   |
|very long |
|text that |
|showing   |
|the buffer|
|          |
|          |
|          |
|          |
|          |
 ----------
size 10x10 cursor 6x0
